api/pkg/service/catalog/syncer.go

```diff
@@ -146,15 +146,15 @@ func (s *syncer) Process() error {
 	log := s.logger.With("action", "process")
 	db := s.db
 
-	job := model.SyncJob{}
+	job := &model.SyncJob{}
 
 	// helper to update job state
 	setJobState := func(s model.JobState) {
 		job.SetState(s)
 		db.Model(&job).Updates(job)
```

Comment:
I think we should use `Model(&model.SyncJob{})` instead of `&job` since it is confusing to read.